pages/getting_started.rst

```diff
@@ -4,7 +4,13 @@
 Getting Started
 ***************
 
-The fastest way to evaluate Graylog is to download the virtual appliance file and run it in VMWare or VirtualBox.
+To get the first look what Graylog is and how you can work with we have a :ref:`virtual appliance <virtual-machine-appliances>` to run on VMWare or Virtualbox. The following will guide you to start using Graylog.
```

Comment:
* a first look
  * work with it
  * that runs on
  * guide you through the first steps of using Graylog